src/include/sof/lib/mailbox.h

```diff
@@ -16,6 +16,16 @@
 #include <stddef.h>
 #include <stdint.h>
 
+/* SRAM window 0 FW "registers" */
+#define SRAM_REG_ROM_STATUS			0x0
+#define SRAM_REG_FW_STATUS			0x4
+#define SRAM_REG_FW_TRACEP			0x8
+#define SRAM_REG_FW_IPC_RECEIVED_COUNT		0xc
+#define SRAM_REG_FW_IPC_PROCESSED_COUNT		0x10
+#define SRAM_REG_FW_TRACEP_SLAVE_CORE_BASE	0x14
+#define SRAM_REG_FW_END \
+	(SRAM_REG_FW_TRACEP_SLAVE_CORE_BASE + (PLATFORM_CORE_COUNT - 1) * 0x4)
```

Comment:
Might be better to define 4 (at least, 8 might be better long term?) slave core slots statically to make sure that subsequent regs have stable, platform independent offset.

Reply:
ok, changed to 8